index.html

```diff
@@ -89,27 +141,94 @@
 <script>
   var data = [];
   var options = {
-    yearRange: 'all'
+    state: 'all'
   };
 
-  var svg;
+  var svg, x , y, histScale;
+  var margin = { top: 15, right: 15, bottom: 40, left: 60 }
+  var width = 840 - margin.left - margin.right;
+  var height = 600 - margin.top - margin.bottom;
 
   // FETCH DATA
 
+  d3.json('party_contr_by_zip.json', function(error, json) {
+    if (error) { throw error; }
+
+    data = json;
+
+    setup();
+    update();
+  });
+
   function setup() {
     // SVG
 
     svg = d3.select('#chart')
-      .append('svg');
+      .append('svg')
+      .attr("width", width + margin.right + margin.left)
+      .attr("height", height + margin.top + margin.bottom)
+      //.style("background", "lightgrey")
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
 
     // SCALES
 
```

Comment:
So, your two scales are using the same units and have a fairly similar data domain. I'd strongly suggest making the scaling equivalent. By this I mean, use the same exact scale for both axes, so that the distance associated with a change along the x or y axis is the same amount of money. That make sense?